### DISCUSSION TITLE
Migrate to dockertest or similar tools

### DISCUSSION THREAD

**Discussion by amanbolat** [go-jet/jet#227] [Ideas]
Hi,

Docker compose is a good approach for testing locally when a manual interaction is needed if any bugs are found. However, the projects seem to be in a good shape to migrate to the tools such as `dockertest` that create the containers inside the tests themselves without spinning up the environment by hands. It will also make tests more declarative in the way that the result and the way of running will be the same in every environment, including the GitHub Workflows.

@go-jet What do you think?

**Comment by go-jet** [comment]
Thanks for the idea, I'll have a look at `dockertest`.  :eyes:
